game_function.php

```diff
@@ -0,0 +1,24 @@
+
+<?php
+
+function printStats(array $result): string
+{
+
+    $game_data = '';
+
+    foreach ($result as $data) {
+        '<div class="game">';
+        $game_data .= '<h3>Game: ' . $data['name'] . '</h3>';
+        '<div class="info">';
+        $game_data .= '<span>Rating: ' . $data['rating'] . '/5' . '</span><br />';
+        $game_data .= '<span>game duration: ' . $data['playing_time'] . ' minutes' . '</span><br />';
+        $game_data .= '<span>minimum players: ' . $data['min_players'] . '</span><br />';
+        $game_data .= '<span>maximum players: ' . $data['max_players'] . '</span><br />';
+        '</div>';
+        '</div>';
```

Comment:
This string doesn't actually get output anywhere. If you wanted it to be part of your HTML for each game you would also need to concatenate it into $game_data